examples/typescript/bulkReassignSchedules.ts

```diff
@@ -0,0 +1,46 @@
+import {LookerNodeSDK} from '@looker/sdk-node'
+import {IScheduledPlan} from '@looker/sdk'
+
+
+const sdk = LookerNodeSDK.init40()
+
+// schedules will either return for all users or for a specific user id if passed
+const getSchedules = async (allUsers: boolean, userID?: number): Promise<IScheduledPlan[]> => {
+
+    let schedules: IScheduledPlan[];
+
+    if (allUsers) {
```

Comment:
Looks like userID is checked lower down

Reply:
do you mean check for one or the other being set? I think an undefined `userID` indicates "all users" and this could be simplified to branch on whether `userID` is provided or not

Reply:
Yeah an undefined userID should indicate fetching schedules for "all_users." Sounds like it might be more straightforward to write this such that we simply check whether userID is undefined/null or not. I can make that change